documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1332,15 +1332,92 @@ boolean.selector=.*MYTABLE.FLAG,.*.IS_ARCHIVED
 
 [id="oracle-decimal-types"]
 === Decimal types
-The setting of the Oracle connector configuration property, `decimal.handling.mode` determines how the connector maps decimal types.
 
+The {prodname} Oracle connector maps decimal types based on the value of the xref:{link-oracle-connector}#oracle-property-decimal-handling-mode[`decimal.handling.mode`] configuration property.
+
+decimal.handling.mode=precise::
 When the `decimal.handling.mode` property is set to `precise`, the connector uses Kafka Connect `org.apache.kafka.connect.data.Decimal` logical type for all `DECIMAL` and `NUMERIC` columns.
 This is the default mode.
++
+.Mappings when `decimal.handing.mode=precise`
+[cols="30%a,15%a,55%a",options="header",subs="+attributes"]
+|===
+|Oracle data type
+|Literal type (schema type)
+|Semantic type (schema name)
+
+|`NUMERIC[(P[,S])]`
+|`BYTES`
+a|`org.apache.kafka.connect.data.Decimal` +
+The `scale` schema parameter contains an integer that represents how many digits the decimal point shifted.
+
+|`DECIMAL[(P[,S])]`
+|`BYTES`
+a|`org.apache.kafka.connect.data.Decimal` +
+The `scale` schema parameter contains an integer that represents how many digits the decimal point shifted.
+
+|`SMALLMONEY`
+|`BYTES`
+a|`org.apache.kafka.connect.data.Decimal` +
+The `scale` schema parameter contains an integer that represents how many digits the decimal point shifted.
+
+|`MONEY`
+|`BYTES`
+a|`org.apache.kafka.connect.data.Decimal` +
+The `scale` schema parameter contains an integer that represents how many digits the decimal point shifted.
```

Comment:
These types are not supported with Oracle AFAIK // CC @Naros.

Reply:
Hi guys, so `SMALLMONEY` and `MONEY` are not valid data types for Oracle.  In fact these are represented most often using other data types such as `DECIMAL`, `FLOAT`, `NUMERIC`, `NUMBER` with appropriate precision and scale values to fit the desired outcome.  

Furthermore, we already include these data types in the Numeric Types section of the docs.  It doesn't make much sense to also list these in the Decimal Types section too.  Is there a logical way we can embed the implied behavior of the `decimal.handling.mode` in the notes section for the respective data types or add a footnote?

Reply:
@Naros Sorry, but I'm a little bit lost here. I'm not sure what you're asking. I get that we should remove the `*MONEY` types, and that those types are typically represented via other data types  that the _Numeric Types_ section already mentions. 
But I'm not clear on what you mean by 
_Is there a logical way we can embed the implied behavior of the `decimal.handling.mode` in the notes section for the respective data types or add a footnote_

Might be good to have a quick call to clarify.

Reply:
What I mean @roldanbob is that we have `NUMERIC` and `DECIMAL` data types listed under Numeric Types and then again under Decimal Types but with varied information based on the `decimal.handling.mode` setting.  I don't think we should be duplicating was my point.  Can we represent this in a way that doesn't involve duplication?

Reply:
Ah yes, good point; each type mapping should be listed only once. Good idea to see how it's done for the other connectors; perhaps make "decimal types" and the different mappings depending on the decimal.handling.mode option a sub-section of "numeric types", and discuss the decimal types only in that sub-section?